### PR TITLE
Better error on ENOENT

### LINKED ISSUE #42: idl publish fails with ENOENT
Looks like we're getting an error when trying to read from cache?

```
idl publish --repository=... --cacheDir=/.../.idl --
ERR: Error: ENOENT, readdir '/...'
```

### PR DESCRIPTION
**Pull request by malandrew**
Closes #42

cc/ @Raynos @kriskowal

### CODE REVIEW

**Comment by Raynos** [comment]
lgtm.

**Comment by blampe** [comment]
+1 much more helpful error

File: test/lib/test-cluster.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ function TestCluster(opts) {
     var self = this;
 
     self.remoteRepos = extend(defaultRepos, opts.remoteRepos || {});
+    // console.log(self.remoteRepos.Z.files.idl['github.com'].org);
```
Comment (blampe) [inline test/lib/test-cluster.js]: Dead code?

**Comment by malandrew** [comment]
I'll remove that line in another commit. I missed it.

**Pull request merged**